src/compenents/Todo.jsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from "react";
+import TodoList from "./TodoList";
+
+const Todo = () => {
+  const [todo, setTodo] = useState("");
+  const [tasks, setTasks] = useState(
```

Comment:
You should consider sticking to one name here. For instance, instead of having `todo` and `tasks` you could choose a single `todo` on line 5 and line 6 becomes `const [tasks, setTasks] = useState(...` becomes `const [todos, setTodos] = ...` or simply go with a single `task` on line 5 and many `tasks` on line 6 to stay consistent in naming.